lib/elixir/src/elixir.erl

```diff
@@ -10,7 +10,9 @@
 -define(system, 'Elixir.System').
 
 %% Top level types
--export_type([char_list/0, struct/0, as_boolean/1, keyword/0, keyword/1]).
+%% TODO: Deprecate in v1.5 - type char_list
```

Comment:
My search patter for deprecations is `TODO: Deprecate` regardless of version and filter results for specific version later.